android/electrode-reactnative-bridge/src/main/java/com/walmartlabs/electrode/reactnative/bridge/RequestHandlerProcessor.java

```diff
@@ -91,7 +91,7 @@ public void onSuccess(TResp obj) {
 
     @Override
     public boolean unregister() {
-        Logger.d(TAG, "Removing registered request handler %s with id %s", handler, id);
+        Logger.w(TAG, "Removing registered request handler %s with id %s", handler, id);
```

Comment:
Is this now an unexpected ("warning") case, or can it stay "debug"?

Reply:
oops. Good catch!. typo.

Reply:
In fact, I will clean up this logic now.